lib/msf/core/post/architecture.rb

```diff
@@ -0,0 +1,52 @@
+# -*- coding: binary -*-
+
+module Msf::Post::Architecture
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Compat' => {
+          'Meterpreter' => {
+            'Commands' => %w[
+              stdapi_railgun_api
+            ]
+          }
+        }
+      )
+    )
+  end
+
+  def get_os_architecture
+    result = get_os_architecture_impl
+    if result.nil?
+    end
```

Comment:
Was this meant to do something? Seems like you should be throwing an error or something here but you aren't. Currently this is dead code that won't do anything.